src/reducers/filter.js

```diff
@@ -0,0 +1,8 @@
+export default (state = 'All', action) => {
+  switch (action.type){
+    case 'CHANGE_FILTER':
+      return action.filter;
+    default:
+      return 'All';
```

Comment:
This should be:

```javascript
default:
  return state
```
Otherwise, ESLint should have told you that `state` in the function's argument wasn't used at all.